Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp

```diff
@@ -101,7 +101,7 @@ class Mesh_3_plugin :
               this, SLOT(mesh_3_surface()));
     }
 
-    actionSplitPolylines = new QAction("Split polylines in a graph", mw);
+    actionSplitPolylines = new QAction("Split polylines in a graph (Tet Mesh)", mw);
```

Comment:
```suggestion
    actionSplitPolylines = new QAction("Split Polylines in a Graph (Tet Mesh)", mw);
```

Reply:
@lrineau could you suggest a better name to Maxime? Build Feature Graph?

Reply:
Yes, `Build Features Graph` is correct, maybe with `for Mesh_3` as a suffix.